app/src/main/res/values/strings.xml

```diff
@@ -791,26 +792,26 @@ How to translate with transifex:
     <string name="custom">Custom</string>
     <string name="set">Set</string>
     <string name="calendar">Calendar</string>
-    <string name="video_call">Video Call</string>
-    <string name="audio_call">Audio Call</string>
+    <string name="video_call">Video call</string>
+    <string name="audio_call">Audio call</string>
     <string name="started_a_call">started a call</string>
     <string name="nc_settings_phone_book_integration_phone_number_dialog_429">Error 429 Too Many Requests</string>
     <string name="nc_caption">Caption</string>
     <string name="languages_error_title">Retrieval failed</string>
     <string name="languages_error_message">Languages could not be retrieved</string>
-    <string name="edit_message_icon_description">Edit Message Icon</string>
+    <string name="edit_message_icon_description">Edit message icon</string>
     <string name="nc_edit_message">Edit</string>
-    <string name="nc_send_edit_message">Send Edit Message</string>
-    <string name="nc_clear_edit_message">Clear Edit Message</string>
+    <string name="nc_send_edit_message">Send edit message</string>
+    <string name="nc_clear_edit_message">Clear edit message</string>
     <string name="edit_error_24_hours_old_message">Cannot Edit Messages older than 24 hours</string>
     <string name="conversation_is_read_only">Conversation is read only</string>
-    <string name="nc_edit_message_text">Edit Message Text</string>
+    <string name="nc_edit_message_text">Edit message text</string>
     <string name="hint_edited_message">(edited)</string>
     <string name="nc_conversation_not_found">Conversation not found</string>
     <string name="add_to_notes">Add to Notes</string>
     <string name="nc_edited_by_admin">Edited by admin</string>
-    <string name="nc_clear_edit_button">Clear Edit Button</string>
-    <string name="nc_edit_icon">Edit Icon</string>
+    <string name="nc_clear_edit_button">Clear edit button</string>
+    <string name="nc_edit_icon">Edit icon</string>
```

Comment:
There is already the "Edit message" and "Edit" alt text, as well as "Cancel editing", these should not be needed? It seems like if you use the screenreader, these could lead to tuplicate reading aloud.

In any case again here the words "icon" and "button" should not be necessary.


```suggestion
    <string name="nc_clear_edit_button">Cancel editing</string>
    <string name="nc_edit_icon">Edit</string>
```